app/assets/javascripts/consumer_role.js

```diff
@@ -11,8 +11,13 @@ $(document).on('change', "#person_no_dc_address, #dependent_no_dc_address, #no_d
 });
 
 $(document).on('ready', function(){
+    if(`EnrollRegistry.feature_enabled?(:ridp_h139) && Rails.env.production? && !(ENV['ENROLL_REVIEW_ENVIRONMENT'] == 'true')`) {
```

Comment:
how is JavaScript able to evaluate this logic?

Reply:
i added it in escape strings

Reply:
the backticks are used for string interpolation or tagged template literals -- they have no affect on how code is evaluated